src/views/citation_editable.ts

```diff
@@ -47,7 +51,7 @@ const createBibliographySection = (node: ManuscriptNode) =>
 export class CitationEditableView extends CitationView<EditableBlockProps> {
   private editor: HTMLElement
   private contextMenu: HTMLElement
-
+  private can: Capabilities = this.props.getCapabilities()
```

Comment:
do you need to specify type, will it not be derived from getCapabilities return type?

Reply:
fixed